### ISSUE TITLE
Keyboard not working with Aurora DX on Asus Vivobook N6505MU

### ISSUE THREAD

**Issue by Haraven** [ublue-os/bluefin#1646]
### Describe the bug

After installing Aurora DX on my laptop (Asus Vivobook N6505MU), I was unable to use the built-in keyboard. It works fine in the installer, when I type into the keyboard text input. But as soon as the OS installs and I get to SDDM, I can't type anything.

### What did you expect to happen?

Of course, my keyboard should work out of the box when installing Ublue Aurora :)

### Output of `rpm-ostree status`

```shell
state: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: no runs since boot
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/aurora-dx-asus-nvidia:latest
                   Digest: sha256:f8acae24e66276c250351c68ff223a633d72d09dd3490fe60d5847fbfa7fa8a1
                  Version: 40.20240902.0 (2024-09-05T04:52:03Z)

  ostree-image-signed:docker://ghcr.io/ublue-os/aurora-dx-asus-nvidia:latest
                   Digest: sha256:4a128003a56ff585759e7fdd49693bd83099da7bbd60ad5b62d4d7b1ad131a31
                  Version: 40.20240830.0 (2024-08-31T04:51:21Z)

```

### Output of `groups`

```shell
shelomoh wheel docker incus-admin lxd libvirt
```

### Extra information or context

Looking this up online, I found out that it's a kernel issue (see [this](https://bugzilla.kernel.org/show_bug.cgi?id=218954)). Something to do with Asus Vivobooks using a non-standard PS/2 keyboard? 

The kernels provided by the Aurora DX images don't have my laptop patched. I tried using both the generic desktop/laptop image with NVIDIA drivers and DX settings, as well as the ASUS-specific image with NVIDIA drivers and DX settings. I'm not even sure if I should use the ASUS image for my Vivobook, since it seems tied to ROG laptops?

Anyway, I can confirm that if I use a different OS that comes with a higher kernel version, the keyboard works fine. On OpenSUSE Tumbleweed, I had kernel 6.10.5 and it worked fine.

On Garuda Linux, initially I had kernel 6.8.7 and the keyboard didn't work, but after plugging in a USB keyboard and updating the kernel to 6.10.7, it worked again.

Aurora seems to ship with 6.9.3 (generic desktop/laptop image) and 6.10 (Asus image), neither of which have my Vivobook N6505MU's keyboard patched.

I understand I can compile the kernel manually, but that seems like a lot of extra work (which might potentially break when updating my system?) just to get a working installation going. It kind of defeats the purpose of the OS being maintenance-free.

**Comment by castrojo** [comment]
Maybe the extra asus enablement in the fsync kernel will help, can you try rebasing to the `latest` tag? 

https://docs.projectbluefin.io/administration#switching-between-channels

**Comment by Haraven** [comment]
I tried it. In short, still no luck.

I no longer had Aurora on my system so I made a separate partition and redownloaded the latest image and installed it.
The keyboard didn't work after installation so I plugged in a USB keyboard and tried running `ujust rebase-helper`, thinking maybe I wasn't on `latest` yet. The command said my image was already on latest.

I tried running `ujust update` again and rebooted, but that didn't change anything..
